websites/L/Lazada/presence.ts

```diff
@@ -0,0 +1,68 @@
+const presence = new Presence({
+		clientId: "1201520071055515698",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let item: HTMLElement;
```

Comment:
It doesn't make sense for this variable to be a global variable considering its usage.